ReactNativeTypeScriptTemplate/app.tsx

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
Compare to https://github.com/forcedotcom/SalesforceMobileSDK-Templates/blob/dev/ReactNativeTemplate/app.js